src/amo/components/Home/index.js

```diff
@@ -252,8 +252,8 @@ export class HomeBase extends React.Component {
         <LandingAddonsCard
           addons={collections[2]}
           className="Home-FeaturedCollection"
-          header={i18n.gettext('Dynamic media downloaders')}
-          footerText={i18n.gettext('See more dynamic media downloaders')}
+          header={i18n.gettext('Trending extensions')}
+          footerText={i18n.gettext('See more trending extensions')}
           footerLink={
             `/collections/${COLLECTIONS_TO_FETCH[2].user}/${COLLECTIONS_TO_FETCH[2].slug}/`
```

Comment:
Should trending extensions be a collection? I guess the programming update would say but I expected this to link to `/search/?sort=hotness&type=extension`

Reply:
Good question. @devaneymoz, do you intend for `Trending extensions` to be populated via a collection, or can we just use the query as suggested above, an example of which would be https://addons-dev.allizom.org/en-US/firefox/search/?sort=hotness&type=extension ?